Sources/Sentry/include/SentrySwizzle.h

```diff
@@ -152,6 +152,10 @@ typedef void (*SentrySwizzleOriginalIMP)(void /* id, SEL, ... */);
 /// The selector of the swizzled method.
 @property (nonatomic, readonly) SEL selector;
 
+#if TEST
+@property (nonatomic) bool originalCalled;
```

Comment:
`m`: To be consistent with the rest of the SDK.
```suggestion
@property (nonatomic) BOOL originalCalled;
```